src/app-layout/drawer/overflow-menu.tsx

```diff
@@ -18,9 +19,9 @@ export default function OverflowMenu({ items, onItemClick, customTriggerBuilder,
     <InternalButtonDropdown
       items={items.map(item => ({
         id: item.id,
-        text: item.ariaLabels?.content || 'Content',
-        iconName: item.trigger.iconName,
-        iconSvg: item.trigger.iconSvg,
+        text: 'drawerName' in item.ariaLabels ? item.ariaLabels?.drawerName : item.ariaLabels?.content || 'Content',
```

Comment:
This is how I'm getting around the different types for ariaLabels and trigger. Let me know if there's a better way to do this but it seems to work quite well.